pkg/links/azure/enricher/storage_account.go

```diff
@@ -36,9 +36,9 @@ func (s *StorageAccountEnricher) Enrich(ctx context.Context, resource *model.Azu
 	// Sanitize the storage account name for URL encoding
 	storageAccountNameForURL := url.QueryEscape(strings.TrimSpace(storageAccountName))
 
-	// Create HTTP client with timeout
+	// Create HTTP client with timeout (reduced from 10s for better performance)
```

Comment:
```suggestion
	// Create HTTP client with timeout
```

No need to mention we reduced from 10s